lib/temjin/card_command/add_command.rb

```diff
@@ -1,8 +1,7 @@
 module Temjin
   class CardCommand
     class AddCommand < Temjin::TrelloAPICommand
-      parameter 'BOARD', 'board name'
-      parameter 'LIST', 'list name'
+      list_parameter
```

Comment:
Does this imply I cannot set a board on the add command if I wanted to override default board or was not using the default board setting?

Reply:
I guess it's not exactly straight forward anymore, but `list_parameter` adds a `-b, --board` option because you need a board to find a list. Think of it as I'm defining the most drilled-down object required, and that object knows what other options it may need.

[source](https://github.com/komidore64/temjin/pull/3/commits/3a39448f35e4aba0aaca49d6ab55869f9c60c7a6#diff-20ffb50fb417f4cf9ab55c274eff6e3bR39) for reference.

Reply:
Thanks for the clarification!